modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/planner/JoinColocationPlannerTest.java

```diff
@@ -111,15 +109,16 @@ public void joinSameTableComplexAff() throws Exception {
     }
 
     /**
-     * Re-hashing based on simple affinity is possible, so bigger table with complex affinity should be sended to the smaller one.
+     * Re-hashing based on simple affinity.
+     *
+     * <p>The table with complex affinity should be sent to the table with simple affinity.
```

Comment:
COMPLEX_TBL is twice of the size of the SIMPLE_TBL, so why do you expect the bigger table being sent to the smaller one?